apps/datahub-e2e/src/e2e/datasets.cy.ts

```diff
@@ -47,32 +47,40 @@ describe('datasets', () => {
       cy.get('@sortBy')
         .getActiveDropdownOption()
         .invoke('attr', 'data-cy-value')
-        .should('equal', 'desc,_score')
+        .should('equal', 'desc,createDate')
     })
   })
 
   describe('display of dataset previews', () => {
-    it('should display a logo for first and a placeholder for second result', () => {
-      cy.get('@sortBy').selectDropdownOption('desc,createDate') // this makes the order reliable
-      cy.get('@sampleResult')
+    it('should display a placeholder for first result and a logo for second result', () => {
```

Comment:
This test has been changed because "sampleResult" dataset (Metadata for testing purposes) had an erroneous image and was not displaying a logo anymore. Besides, the new sorting is also reliable because the favorites are cleared before the e2e are run.